src/Login/LoginForm.tsx

```diff
@@ -0,0 +1,83 @@
+import React, {useState} from 'react'
+import PasswordToggle from "../common/PasswordToggle";
+import {post} from "../utils/http";
+
+type LogInfo = {
+    email: string,
+    password: string
+}
+
+type Props = {
+    whereTo: string
+}
+
+function LoginForm(props: Props){
+
+    const [PasswordInputType, ToggleIcon] = PasswordToggle();
+    const [logInfo, setLogInfo] = useState<LogInfo>({email: "", password: ""});
```

Comment:
Probably is better to have 2 different states here, one for the email and the other for the password, this way you avoid the check on the `changeInfo` method, and it's also more simple to handle them.